cont3xt/vueapp/src/components/pages/Help.vue

```diff
@@ -93,11 +93,29 @@
 
       <!-- integrations -->
       <div v-if="visibleTab === 'integrations'">
-        <div class="text-center">
-          <b-card>
-            <h1>Coming soon!</h1>
-          </b-card>
-        </div>
+        <h3>
+          <span class="fa fa-fw fa-key mr-2"></span>
+          Integrations
+        </h3>
+        <p>
+          You must use this page to configure your integations.
+          Most integrations require API keys (and some require more data) in order to use them.
+          Integrations are configured per user, not per Cont3xt application.
```

Comment:
you can do global config in the config file and then we show a globe. this will over ride it. we should mention that